content/contributors/projects/_index.md

```diff
@@ -325,7 +325,7 @@ CloudEvents Specification
 *"Open source Kubernetes native workflows, events, CI and CD"* - [https://argoproj.github.io/ ][$Argo overview]
 
 -	**Project Repository:** https://github.com/argoproj/
--	**Contributor Guide:** [argo/community/][(https://github.com/argoproj/argo/tree/master/community#contributing-to-argo\)
+-	**Contributor Guide:** [argo/community/](https://github.com/argoproj/community-contrib-docs/blob/3b76affa356dbc3160b31f9a96ac81dfa49017ae/CODE_OF_CONDUCT.md)
```

Comment:
```suggestion
-	**Contributor Guide:** [argo/community/](https://github.com/argoproj/community-contrib-docs/blob/master/argo-cd/developer/contributing.md)
```

That links to an old commit of their code of conduct, it looks like their dev docs have moved into the argo-cd/developer dir. I think my suggestion si the right one 👍